Packs/Keyfactor/Integrations/KeyfactorDev/KeyfactorDev.yml

```
@@ -0,0 +1,122 @@
+category: Network Security
+commonfields:
+  id: Keyfactor Dev
+  version: -1
+configuration:
+- defaultvalue: https://fis.thedemodrive.com/KeyfactorAPI/
+  display: API URL
+  name: host
+  required: true
+  type: 0
+- display: Username
+  name: username
+  required: true
+  type: 4
+- display: Password
+  name: password
+  required: true
+  type: 4
```

```suggestion
- display: Username
  name: credentials
  required: true
  type: 9
```
Use credential object instead (`"type": 9`).
In the python code you may access it as follows:
```python
params = demisto.params()
username = params.get('credentials', {}).get('identifier')
password = params.get('credentials', {}).get('password')